### PR TITLE
ESSNTL 5248 update mapped system with groups in archiver

### PR DESCRIPTION
**Pull request by petrblaho**
ESSNTL-5248

- chore: remove renaming of listener_metrics module
- refactor: change name and signature of processing method in archiver
- feat: calls baselines' update associated systems when system do check-in

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [x] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by petrblaho** [comment]
Needs https://github.com/RedHatInsights/kerlescan/pull/103 merged and new kerlescan tag updated here.

File: historical_system_profiles/archiver.py [hunk 1]
```diff
@@ -125,33 +125,35 @@ def _archive_profile(data, ptc, logger, notification_service):
...
-    system_check_in,
-    display_name,
-    tags,
-    notification_service,
-    service_auth_key,
-    logger,
+def _further_processing(
+    inventory_id=None,
+    account_id=None,
+    org_id=None,
```
Comment (vkrizan) [inline historical_system_profiles/archiver.py:151]: I'd recommend to have these three mandatory. That can be done by not specifying their default values.

**Review by vkrizan — APPROVED** [review]

**Pull request merged**